README.md

```diff
@@ -22,85 +52,77 @@ Soft Forks & protocol overview|IsSuperMajority signaling|[BIP 65](https://github
 Cryptography|zero knowledge proofs|[Introduction to SNARKs](https://www.youtube.com/watch?v=jr95o_k_SwI&feature=youtu.be)|
-Cryptography|Bulletproofs|[How Bulletproofs Could Make Bitcoin Privacy Less Costly](https://bitcoinmagazine.com/articles/how-bulletproofs-could-make-bitcoin-privacy-less-costly/), [Bulletproofs: Faster Rangeproofs and Much More](https://blockstream.com/2018/02/21/bulletproofs-faster-rangeproofs-and-much-more/), [Bulletpoints on bulletproofs](https://joinmarket.me/blog/blog/bulletpoints-on-bulletproofs/), [From Zero Knowledge Proofs to Bulletproofs Paper](https://joinmarket.me/blog/blog/from-zero-knowledge-proofs-to-bulletproofs-paper/),[Building on Bulletproofs](https://medium.com/@cathieyun/building-on-bulletproofs-2faa58af0ba8)|
-Cryptography|Commitment schemes; pedersen commitments|[Cryptography](https://www.adjoint.io/docs/cryptography.html#commitment-schemes), [Non-Interactive and Information- Theoretic Secure Verifiable Secret Sharing](https://www.cs.cornell.edu/courses/cs754/2001fa/129.PDF)|
-Cryptography|Schnorr| [Simple Schnorr Multi-Signatures with Applications to Bitcoin](https://eprint.iacr.org/2018/068.pdf), [Liars, cheats, scammers and the Schnorr signature](https://joinmarket.me/blog/blog/liars-cheats-scammers-and-the-schnorr-signature/)|
+Cryptography|Bulletproofs|[How Bulletproofs Could Make Bitcoin Privacy Less Costly](https://bitcoinmagazine.com/articles/how-bulletproofs-could-make-bitcoin-privacy-less-costly/), [Bulletproofs: Faster Rangeproofs and Much More](https://blockstream.com/2018/02/21/bulletproofs-faster-rangeproofs-and-much-more/), [Bulletpoints on bulletproofs](https://joinmarket.me/blog/blog/bulletpoints-on-bulletproofs/), [From Zero Knowledge Proofs to Bulletproofs Paper](https://joinmarket.me/blog/blog/from-zero-knowledge-proofs-to-bulletproofs-paper/), [Building on Bulletproofs](https://medium.com/@cathieyun/building-on-bulletproofs-2faa58af0ba8)|
+Cryptography|Commitment schemes; pedersen commitments|[Commitment schemes](https://www.adjoint.io/docs/cryptography.html#commitment-schemes), [Non-Interactive and Information- Theoretic Secure Verifiable Secret Sharing](https://www.cs.cornell.edu/courses/cs754/2001fa/129.PDF)|
+Cryptography|Schnorr| [Introduction to Schnorr Signatures (video)](https://youtu.be/XKatSGCZ-gE), [Simple Schnorr Multi-Signatures with Applications to Bitcoin](https://eprint.iacr.org/2018/068.pdf), [Liars, cheats, scammers and the Schnorr signature](https://joinmarket.me/blog/blog/liars-cheats-scammers-and-the-schnorr-signature/)|
 Cryptography|Diffie-Hellman|[Diffie-Hellman Key Exchange: A Non-mathematician’s explanation](http://academic.regis.edu/cias/ia/palmgren_-_diffie-hellman_key_exchange.pdf)|
```

Comment:
http://academic.regis.edu/cias/ia/palmgren_-_diffie-hellman_key_exchange.pdf does not load for me right now

Reply:
```suggestion
Cryptography|Diffie-Hellman|[Diffie-Hellman Key Exchange: A Non-mathematician’s explanation](https://web.archive.org/web/20171013073858/http://academic.regis.edu/cias/ia/palmgren_-_diffie-hellman_key_exchange.pdf)|
```

Reply:
Found another resource